backend/core/views.py

```
@@ -607,6 +649,56 @@ class RiskMatrixViewSet(BaseModelViewSet):
     def colors(self, request):
         return Response({"results": get_risk_color_ordered_list(request.user)})
 
+    @action(detail=False, name="Get risk level choices")
+    def risk(self, request):
+        viewable_matrices: list[RiskMatrix] = RoleAssignment.get_accessible_object_ids(
```

Ruff (0.8.2)

`RiskMatrix` may be undefined, or defined from star imports

(F405)